tools/analyzers/longfile/longfile.go

```diff
@@ -2,53 +2,114 @@ package longfile
 
 import (
 	_ "embed"
+	"flag"
 	"fmt"
 	"go/ast"
 	"go/token"
+	"strconv"
 	"strings"
 
 	"golang.org/x/tools/go/analysis"
 )
 
-var max int
-var exceptions map[string]int
+var max = 1500
```

Comment:
This should be a const

Reply:
Thanks @iholder-redhat. Fixed. Done more than that: add two parameters; one for regular go files (default = 1000 lines) and one for test files (default = 1500 lines).

Reply:
Awesome! Thanks!